main.go

```diff
@@ -167,26 +167,41 @@ Generate an identity seed and launch a gateway:
 			EnvVars: []string{"RAINBOW_MAX_FD"},
 			Usage:   "Maximum number of file descriptors. Defaults to 50% of the process' limit",
 		},
+		&cli.StringSliceFlag{
+			Name:    "routing-v1-endpoints",
+			Value:   cli.NewStringSlice(cidContactEndpoint),
+			EnvVars: []string{"RAINBOW_ROUTING_V1_ENDPOINTS"},
+			Usage:   "Routing V1 endpoints to use for routing (comma-separated)",
```

Comment:
:thought_balloon: Are we switching to longer `routing-v1-endpoints` to be consistent with someguy, or just to start conversation? I don't feel strongly, but we could go with shorter `http-routers`

```suggestion
			Name:    "routing-v1-endpoints",
			Value:   cli.NewStringSlice(cidContactEndpoint),
			EnvVars: []string{"RAINBOW_ROUTING_V1_ENDPOINTS"},
			Usage:   "HTTP servers with /routing/v1 endpoints to use for delegated routing (comma-separated)",
```

Reply:
Updated to `http-routers`. Seems like a good idea. Maybe we should update it in Someguy too.